From andrea.gelmini@gelma.net  Fri Feb 26 16:41:50 2010
From: Andrea Gelmini <andrea.gelmini@gelma.net>
Date: Fri, 26 Feb 2010 17:36:57 +0100
Subject: Staging: rt3070: md4.h: Checkpatch cleanup
To: gregkh@suse.de
Cc: devel@driverdev.osuosl.org, bzolnier@gmail.com, Andrea Gelmini <andrea.gelmini@gelma.net>
Message-ID: <1267202220-15640-2-git-send-email-andrea.gelmini@gelma.net>


WARNING: space prohibited between function name and open parenthesis '('
+void MD4Init (MD4_CTX *);

WARNING: space prohibited between function name and open parenthesis '('
+void MD4Update (MD4_CTX *, u8 *, UINT);

WARNING: space prohibited between function name and open parenthesis '('
+void MD4Final (u8 [16], MD4_CTX *);

ERROR: do not use C99 // comments
+#endif //__MD4_H__

Signed-off-by: Andrea Gelmini <andrea.gelmini@gelma.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt3070/md4.h |    7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

--- a/drivers/staging/rt3070/md4.h
+++ b/drivers/staging/rt3070/md4.h
@@ -35,8 +35,8 @@ typedef	struct	_MD4_CTX_	{
 	u8	buffer[64];      /* input buffer */
 }	MD4_CTX;
 
-void MD4Init (MD4_CTX *);
-void MD4Update (MD4_CTX *, u8 *, UINT);
-void MD4Final (u8 [16], MD4_CTX *);
+void MD4Init(MD4_CTX *);
+void MD4Update(MD4_CTX *, u8 *, UINT);
+void MD4Final(u8 [16], MD4_CTX *);
 
-#endif //__MD4_H__
\ No newline at end of file
+#endif /*__MD4_H__*/
